lib/applicants/validation.ts

```diff
@@ -27,7 +27,14 @@ export const requestPermitHolderInformationSchema = object({
   phone: string()
     .required('Please enter a valid phone number')
     .max(12, 'Please enter a valid phone number in the format 000-000-0000'),
-  receiveEmailUpdates: boolean().required(),
+  receiveEmailUpdates: boolean()
+    .nullable()
+    .default(null)
+    .when('type', {
+      is: 'NEW',
```

Comment:
Should be required for both new and renewal applications

Reply:
Just finished rebasing -- I noticed that this changed to just be `receiveEmailUpdates: boolean().required(),` given this came from staging I'm assuming it should be kept that way?

Reply:
Yes!